docs/managed-datahub/slack/saas-slack-troubleshoot.md

```diff
@@ -0,0 +1,69 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# Troubleshoot Slack Issues
+
+<FeatureAvailability saasOnly />
+
+## Verifying the Slack Integration is working
```

Comment:
how about adding some intros before the main paragraph? It seems like the first part is about testing (pre-requisites), and the 2nd/3rd parts are more of a troubleshooting reference... I think organizing these concepts as an intro could help users understand.

e.g. This document provides troubleshooting guidance for Slack setup. For more details on Slack integrations, click here (and let's include links to the slack setup docs)

Reply:
truee